daisy/step_create_networks.go

```diff
@@ -0,0 +1,71 @@
+//  Copyright 2017 Google Inc. All Rights Reserved.
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+
+package daisy
+
+import (
+	"context"
+	"sync"
+)
+
+// CreateNetworks is a Daisy CreateNetwork workflow step.
+type CreateNetworks []*Network
+
+func (c *CreateNetworks) populate(ctx context.Context, s *Step) dErr {
+	var errs dErr
+	for _, n := range *c {
+		errs = addErrs(errs, n.populate(ctx, s))
+	}
+	return nil
```

Comment:
return errs?

Reply:
You can't err out if you don't return them.
![roll-safe-think-about-it](https://user-images.githubusercontent.com/1885442/35360807-d4f9e318-0113-11e8-96d3-7930c7c6b45a.jpg)